src/world/entity/mod.rs

```diff
@@ -591,6 +707,10 @@ impl Player {
         self.common.job_id = job_id;
     }
 
+    pub fn set_sex(&mut self, sex: Sex) {
+        self.common.sex = sex;
+    }
+
```

Comment:
I noticed that `set_sex` and `get_sex` are not used by anything, does it maybe make sense to remove them for now?